static/templates/admin/runEnhancementsForm.html

```diff
@@ -0,0 +1,19 @@
+<form id="run_enhancements_form"class="edit_form">
+	<p>
+		All appropriate enhancements will be performed on the selected objects and their children
+	</p>
+	
+	<span>
+		<input type="checkbox" id="run_enhancements_force" name="run_enhancements_force" /><label for="run_enhancements_force">Force enhancement of objects which have been previously processed?</label>
```

Comment:
I agree that's clearer, but the only problem is that it will also run enhancements for the first time if they had not run previously